bin/helpers/helper.js

```diff
@@ -0,0 +1,292 @@
+
+const logger = require("../helpers/logger").winstonLogger;
+const utils = require('../helpers/utils');
+const fs = require('fs');
+const path = require('path');
+const http = require('http');
+const https = require('https');
+const request = require('request');
+var gitLastCommit = require('git-last-commit');
```

Comment:
Any reason for this being `var`?

Reply:
I had copied thos from O11y code, var isn't required, tested with const, done.